src/python/pants/notes/2.3.x.md

```diff
@@ -2,6 +2,16 @@
 
 See https://www.pantsbuild.org/v2.3/docs/release-notes-2-3 for an overview of the changes in this release series.
 
+## 2.3.0.dev3 (Feb 2, 2021)
+
+### New Features
+
+* Add `--remote-auth-plugin` ([#11503](https://github.com/pantsbuild/pants/pull/11503))
+
+### Plugin API Changes
+
+* Upgrade packaging from 20.8 to 20.9 ([#11508](https://github.com/pantsbuild/pants/pull/11508))
```

Comment:
probably wrong section.

Reply:
Where else would it go? There's an argument for internal-only, i.e. left off.